endoreg_db/management/commands/create_model_meta_from_huggingface.py

```
@@ -0,0 +1,101 @@
+
+    def handle(self, *args, **options):
+        model_id = options["model_id"]
+        model_name = options["model_name"]
+        labelset_name = options["labelset_name"]
+        version = options["meta_version"]
+
+        self.stdout.write(f"Downloading model {model_id} from Hugging Face...")
+
+        try:
+            # Download the model weights
+            weights_path = hf_hub_download(repo_id=model_id, filename="pytorch_model.bin", local_dir="/tmp")
```

Ruff (0.14.2)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)